src/hotspot/share/precompiled/precompiled.hpp

```diff
@@ -26,52 +26,38 @@
 // --disable-precompiled-headers to configure.
 
 // These header files are included in at least 130 C++ files, as of
-// measurements made in November 2018. This list excludes files named
-// *.inline.hpp, since including them decreased build performance.
+// measurements made in August 2025.
 
-#include "classfile/classLoaderData.hpp"
+#include "asm/assembler.hpp"
+#include "asm/macroAssembler.hpp"
 #include "classfile/javaClasses.hpp"
-#include "classfile/systemDictionary.hpp"
+#include "classfile/vmSymbols.hpp"
 #include "gc/shared/collectedHeap.hpp"
-#include "gc/shared/gcCause.hpp"
+#include "interpreter/interpreter.hpp"
 #include "logging/log.hpp"
-#include "memory/allocation.hpp"
+#include "logging/logStream.hpp"
+#include "memory/allStatic.hpp"
+#include "memory/allocation.inline.hpp"
 #include "memory/iterator.hpp"
-#include "memory/memRegion.hpp"
 #include "memory/resourceArea.hpp"
 #include "memory/universe.hpp"
-#include "nmt/memTracker.hpp"
-#include "oops/instanceKlass.hpp"
 #include "oops/klass.hpp"
 #include "oops/method.hpp"
-#include "oops/objArrayKlass.hpp"
-#include "oops/objArrayOop.hpp"
-#include "oops/oop.hpp"
-#include "oops/oopsHierarchy.hpp"
+#include "oops/oop.inline.hpp"
 #include "runtime/atomic.hpp"
+#include "runtime/frame.inline.hpp"
 #include "runtime/globals.hpp"
-#include "runtime/handles.hpp"
+#include "runtime/handles.inline.hpp"
+#include "runtime/interfaceSupport.inline.hpp"
 #include "runtime/javaThread.hpp"
-#include "runtime/mutex.hpp"
-#include "runtime/orderAccess.hpp"
+#include "runtime/mutexLocker.hpp"
 #include "runtime/os.hpp"
-#include "runtime/timer.hpp"
+#include "runtime/sharedRuntime.hpp"
+#include "runtime/stubRoutines.hpp"
 #include "utilities/align.hpp"
-#include "utilities/bitMap.hpp"
-#include "utilities/copy.hpp"
 #include "utilities/debug.hpp"
-#include "utilities/exceptions.hpp"
 #include "utilities/globalDefinitions.hpp"
 #include "utilities/growableArray.hpp"
 #include "utilities/macros.hpp"
 #include "utilities/ostream.hpp"
-#include "utilities/ticks.hpp"
-
-#ifdef TARGET_COMPILER_visCPP
```

Comment:
All of the reported testing was on Linux. These were included specifically because measurements
said their inclusion here was beneficial. And my recollection from previous discussions is that
Visual Studio may be the compiler where precompiled headers are most beneficial.